tests/core/simulation_generator.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+
+import random
+from openfisca_core.simulations import Simulation
+from openfisca_core.periods import period as make_period
+
+
+def make_simulation(tax_benefit_system, nb_persons, nb_groups, **kwargs):
+    """
+        Generate a simulation containing nb_persons persons spread in nb_groups groups
```

Comment:
Would an example (or a link to one) would be useful for fist time users ?

Reply:
Added an example